src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -263,7 +263,10 @@
         "^", SymbolSegment, name="binary_xor", type="binary_operator"
     ),
     LikeOperatorSegment=NamedParser(
-        "like_operator", SymbolSegment, name="like_operator", type="comparison_operator"
+        "like_operator",
+        SymbolSegment,
+        name="like_operator",
+        type="comparison_operator",
```

Comment:
Let's revert this change since nothing actually changed here now?

```suggestion
        "like_operator", SymbolSegment, name="like_operator", type="comparison_operator"
```